src/internal/model/registerDomainModel.ts

```diff
@@ -14,7 +14,7 @@ import { RequestFile } from './models';
 
 export class RegisterDomainModel {
     'domain': string;
-    'subdomain': string;
+    'subdomain'?: string;
```

Comment:
Keep this as-is, we don't want to change `internal` models and subdomain should be a needed entry